mdstat_test.go

```diff
@@ -13,24 +13,39 @@
 
 package procfs
 
-import (
-	"testing"
-)
+import "testing"
+
+func TestMdadm(t *testing.T) {
+	mountPoint := "fixtures/proc"
+	fs, errFs := NewFS(mountPoint)
+
```

Comment:
Why not just use the existing `getProcFixtures(t)` here instead?  If you want a better error message, feel free to change `getProcFixtures(t)` directly.

Reply:
I somehow forgot it existed. I'll use that instead, and change the error message there.